label_studio/data_manager/api.py

```diff
@@ -202,23 +203,28 @@ def get(self, request):
             self.check_object_permissions(request, project)
         else:
             return Response({'detail': 'Neither project nor view id specified'}, status=404)
-
         # get prepare params (from view or from payload directly)
         prepare_params = get_prepare_params(request, project)
         queryset = self.get_task_queryset(request, prepare_params)
         context = self.get_task_serializer_context(self.request, project)
 
+        # get counters
+        total_predictions = Prediction.objects.filter(task_id__in=queryset).count()
```

Comment:
It's important to return null if this condition is not executed - 

return Response({
                "total_annotations": null,
                "total_predictions": null,
                "total": null,
                "tasks": serializer.data,
            })

frontend won't display these counters at all. Otherwise there will be zeros, it's incorrect.

Reply:
Fixed

Reply:
Fixed